app/com/arpnetworking/rollups/RollupGenerator.java

```diff
@@ -70,11 +74,11 @@ public Receive createReceive() {
     /**
      * RollupGenerator actor constructor.
      *
-     * @param configuration play configuration
+     * @param configuration    play configuration
```

Comment:
nit: pretty sure we don't do column alignment on parameter descriptions.

Reply:
Fixed